liquibase-service/src/main/resources/db/rdb/routines/007-sp_s_pagebuilder_postprocessing-002.sql

```diff
@@ -0,0 +1,1520 @@
+CREATE OR ALTER PROCEDURE dbo.sp_s_pagebuilder_postprocessing
+    @Batch_id bigint,
+    @phc_id bigint,
+    @rdb_table_name varchar(250) = 'D_INV_ADMINISTRATIVE',
+    @category varchar(250) = 'INV_ADMINISTRATIVE',
+    @debug bit = 'false'
+    AS
+BEGIN
+	DECLARE @RowCount_no int;
+	DECLARE @Proc_Step_no float= 0;
+	DECLARE @Proc_Step_Name varchar(200)= '';
+	DECLARE @batch_start_time datetime2(7)= NULL;
+	DECLARE @batch_end_time datetime2(7)= NULL;
+
+BEGIN TRY
+SET @Proc_Step_no = 1;
+		SET @Proc_Step_Name = 'SP_Start';
+
+BEGIN TRANSACTION;
+INSERT INTO [dbo].[job_flow_log]( batch_id, [Dataflow_Name], [package_Name], [Status_Type], [step_number], [step_name], [row_count] )
+VALUES( @Batch_id, @category, 'S_'+@category, 'START', @Proc_Step_no, @Proc_Step_Name, 0 );
+
+COMMIT TRANSACTION;
+
+SELECT @batch_start_time = batch_start_dttm, @batch_end_time = batch_end_dttm
+FROM [dbo].[job_batch_log]
+WHERE status_type = 'start' AND
+    type_code = 'MasterETL';
+
+---******************************************************************
+---** START PROCESSING TEXT BASED QUESTIONS AND ANSWERS *************
+
+BEGIN TRANSACTION;
+SET @Proc_Step_no = 2;
+SET @Proc_Step_Name = ' Creating nrt_page temporary table to hold latest answers for act_uid: '+cast(@phc_id as VARCHAR(50));
+
+IF OBJECT_ID('#NRT_PAGE', 'U') IS NOT NULL
+BEGIN
+DROP TABLE #NRT_PAGE;
+END;
+
+SELECT
+    act_uid,
+    nbs_case_answer_uid,
+    nbs_ui_metadata_uid,
+    nbs_rdb_metadata_uid,
+    nbs_question_uid,
+    rdb_table_nm,
+    rdb_column_nm,
+    answer_txt,
+    answer_group_seq_nbr,
+    investigation_form_cd,
+    unit_value,
+    question_identifier,
+    data_location,
+    question_label,
+    other_value_ind_cd,
+    unit_type_cd,
+    mask,
+    data_type,
+    question_group_seq_nbr,
+    code_set_group_id,
+    block_nm,
+    last_chg_time,
+    record_status_cd
+INTO
+    #NRT_PAGE
+FROM
+    dbo.nrt_page_case_answer AS nrt_pca WITH(NOLOCK)
+WHERE
+    nrt_pca.act_uid = @phc_id
+  and nrt_pca.last_chg_time = (
+    select
+    max(last_chg_time)
+    from
+    dbo.nrt_page_case_answer
+    where
+    act_uid = @phc_id
+    );
```

Comment:
Please consider adopting the proposed formatting style to enhance code readability (formatting and minimizing unnecessary line breaks):
```
WHERE 
    nrt_pca.act_uid = @phc_id
  AND nrt_pca.last_chg_time = (
      SELECT max(last_chg_time)
      FROM dbo.nrt_page_case_answer
      WHERE act_uid = @phc_id
  );
  ```